### PR TITLE
Refactoring vue styled

### PR DESCRIPTION
**Pull request by pascalgrimaud**
- fix https://github.com/jhipster/jhipster-lite/issues/1656

### CODE REVIEW

File: src/main/webapp/app/springboot/primary/generator/vue-generator/VueGenerator.vue [hunk 1]
```diff
@@ -1,26 +1,6 @@
...
-        </span>
-      </label>
-    </div>
-    <GeneratorButtonVue
-      :label="'Generate Vue.js'"
-      :service="isVueWithStyle ? 'vue-styled' : 'vue'"
-      :selector-prefix="selectorPrefix"
-      @click.prevent="addVue"
-    />
+    <GeneratorButtonVue :label="'Generate Vue.js'" :service="'vue'" :selector-prefix="selectorPrefix" @click.prevent="addVue" />
```
[the remark this answers is not in the document]
Comment (pascalgrimaud) [reply src/main/webapp/app/springboot/primary/generator/vue-generator/VueGenerator.vue:3]: sorry, I don't understand @seraphinandrieux : can you provide the code you have in mind plz ?

[the remark this answers is not in the document]
Comment (seraphinandrieux) [reply src/main/webapp/app/springboot/primary/generator/vue-generator/VueGenerator.vue:3]:
```suggestion
    <GeneratorButtonVue label="Generate Vue.js" service="vue" :selector-prefix="selectorPrefix" @click.prevent="addVue" />
```

[the remark this answers is not in the document]
Comment (seraphinandrieux) [reply src/main/webapp/app/springboot/primary/generator/vue-generator/VueGenerator.vue:3]: Most of the time we use dynamic syntax (with `:` as prefix) to use a variable exposed by the component. Here you don't have a variable/method to build the string passed. So you don't need to use this syntax.

[the remark this answers is not in the document]
Comment (pascalgrimaud) [reply src/main/webapp/app/springboot/primary/generator/vue-generator/VueGenerator.vue]: thanks :)

**Pull request merged**